infra/docker/Dockerfile

```
@@ -1,42 +1,43 @@
-    url="https://github.com/cosmtrek/air/releases/download/${AIR_VERSION}/air_${AIR_VERSION#v}_linux_amd64.tar.gz"; \
-    sha256="3322883bd2c12c56ac7df605bd4ad9d9f99ca71c8e8d3024a73c28d47a365491"; \
-    wget -qO /tmp/air.tgz "${url}" && \
-    echo "${sha256}  /tmp/air.tgz" | sha256sum -c - && \
+RUN wget -qO /tmp/air.tgz "https://github.com/cosmtrek/air/releases/download/${AIR_VERSION}/air_${AIR_VERSION#v}_linux_amd64.tar.gz" && \
     tar -C /usr/local/bin -xzf /tmp/air.tgz air && \
     rm /tmp/air.tgz
 
-COPY . .
-RUN go build -o main .
+# Install build tools and wget for healthcheck
+RUN apk --no-cache add git wget
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)